UmbracoFood/Controllers/Api/OrderApiController.cs

```diff
@@ -99,7 +99,34 @@ public void PutEditOrder(EditOrderViewModel model)
             {
                 orderService.ChangeStatus(model.OrderId, (OrderStatus) model.Status);
             }
+        }
+
+        [HttpDelete]
+        public void DeleteOrder(int id)
+        {
+            if (id < 1)
+            {
+                throw new Exception("ID nie może być niższe niż 1");
+            }
+
+            var order = orderService.GetOrder(id);
+            if (order == null)
+            {
+                throw new Exception("Zamówienie nie istnieje");
+            }
+
+            orderService.RemoveOrder(id);
```

Comment:
nie lepiej to dać w try? i olać to pobieranie ordera? wylap odpowiedni wyjątek i będzie tak samo bez dodatkowego strzału do bazy